package.json

```diff
@@ -7,15 +7,17 @@
         "app/*",
         "client/**",
         "core/*",
-        "server/*"
+        "server/*",
+        "server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming"
     ],
     "scripts": {
         "prepare": "husky install .husky",
         "lint": "eslint app/ client/ core/ script/ server/ --ext .ts,.tsx && prettier --check .",
         "format": "prettier --write .",
         "format-staged": "npx pretty-quick --staged",
         "clean": "ts-node ./script/clean.ts",
-        "compile": "tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "compile": "npm run steamingClient &&  tsc --build --verbose && npm run postcompile --workspaces --if-present",
+        "steamingClient": "npm run build -w server/aws-lsp-codewhisperer/src.gen/@amzn/codewhisperer-streaming",
```

Comment:
Can this build scripts be contained in the streaming client package and conform with setup on this monorepo? I'd prefer us to make packages introduced consistent with whole project setup, instead of introducing custom setup like this.

Reply:
I think this is needed to ensure that the streaming client gets built before everything else given others will depend on it

Reply:
Looking at `compile` script, currently it does not trigger `compile` script in nested workspaces, only `postcompile` script does that, so whole project is not compiled by running `npm run compile` at root level.
Also I think just enabling `npm run compile -w` can become problematic, as npm does not build dependencies tree for workspaces, according to documentation.

But as you mentioned in thread above, if `codewhisperer-streaming` client is only meant to be used in `aws-lsp-codewhisperer` server, we don't need to expose it and it's commands at root level of this monorepo. Can this commands be moved to that package then?